graphene/types/base.py

```diff
@@ -43,6 +43,7 @@ def __init_subclass_with_meta__(
             return
         _meta.name = name or cls.__name__
         _meta.description = description or trim_docstring(cls.__doc__)
+        _meta.extra = _kwargs
```

Comment:
I think the `extra` field should be an explicit class attribute of the `BaseOptions` class so that it can be properly documented. Otherwise users will be confused about the meaning or the purpose/won't find it at all.